jbehave-support-core/src/main/java/org/jbehavesupport/core/rest/RestServiceHandler.java

```diff
@@ -125,6 +127,43 @@ public void sendRequest(String urlPath, HttpMethod requestMethod, ExamplesTable
         } catch (HttpClientErrorException | HttpServerErrorException e) {
             storeResponse(e);
         }
+        isSuccessful();
```

Comment:
That method is called from "sending" step, why do you perform some implicit verification? What if I would like to sent some invalid request and check 4xx code, in that case the isSuccessful verification will be called also.

Reply:
In that case do not implement success check as status based in success handler method.

Reply:
It is intentionally in sending step. User can specify how to assess answer validity - must contain or miss some headers, etc. Or leave blank, if there is none common behavior for particular API.